azurelinuxagent/common/utils/extensionprocessutil.py

```diff
@@ -103,7 +103,7 @@ def format_stdout_stderr(stdout, stderr):
     """
     Format stdout and stderr's output to make it suitable in telemetry.
     The goal is to maximize the amount of output given the constraints
-    of telemetry.
+    of telemetry. too-few-public-methods<R0903> Disabled: This class is used as an Enum
```

Comment:
Are you missing `# pylint: disable=` here?

Reply:
I think this was a typo, removed